### PR TITLE
(maint) Update e2e tests to be fixed or marked as broken

### PR DESCRIPTION
**Pull request by AdmiringWorm**
## Description Of Changes

This pull request updates the failing e2e tests to either be marked as broken, or to fix the actual tests that we are able to fix at this moment.
The side by side tests has also been removed as we no longer will support this feature going forward.
THe tests that are marked as broken has a task associated with them internally for work that needs to be done.

## Motivation and Context

To know the current status of things, and make our internally running instance of test kitchen succeed.

## Testing

1. Ran through test kitchen.

### Operating Systems Testing

N/A

## Change Types Made

* [ ] Bug fix (non-breaking change).
* [ ] Feature / Enhancement (non-breaking change).
* [ ] Breaking change (fix or feature that could cause existing functionality to change).
* [ ] Documentation changes.
* [ ] PowerShell code changes.
* [x] Tests

## Change Checklist

* [ ] Requires a change to the documentation.
* [ ] Documentation has been updated.
* [ ] Tests to cover my changes, have been added.
* [ ] All new and existing tests passed?
* [ ] PowerShell code changes: PowerShell v2 compatibility checked?

## Related Issue

https://app.clickup.com/t/20540031/PROJ-309

### CODE REVIEW

File: tests/chocolatey-tests/commands/choco-install.Tests.ps1
Comment (AdmiringWorm) [reply tests/chocolatey-tests/commands/choco-install.Tests.ps1:537]: I removed it as the behavior is different between licensed and FOSS, we should look at making the behavior the same for both.
To prevent that being ignored again, I removed the FossOnly flag.

File: tests/chocolatey-tests/commands/choco-pack.Tests.ps1
Comment (AdmiringWorm) [reply tests/chocolatey-tests/commands/choco-pack.Tests.ps1:201]: Yes, it was marked as PartiallyBroken on purpose. The skipped values are the ones that are broken, but the rest of the invalid failures are still valid.

**Review by corbob — APPROVED** [review]
These look good to me. I'll keep an eye on the Team City build, to merge this after that's completed.

**Comment by corbob** [comment]
There were a few failures on Test Kitchen. I've pushed up fixes to all of them, and am running a fresh set of tests. Once they complete I'll get this merged in :D

**Review by corbob — APPROVED** [review]

**Comment by corbob** [comment]
Thanks for getting these sorted @AdmiringWorm 👍

**Pull request merged**
